petergreve/week2/Ass3/index.html

```diff
@@ -0,0 +1,233 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<title>Deep dive into React</title>
+	<script src="http://fb.me/react-0.14.7.js"></script>
+	<script src="http://fb.me/react-dom-0.14.7.js"></script>
+	<script src="https://cdnjs.cloudflare.com/ajax/libs/babel-core/5.8.23/browser.min.js"></script>
+</head>
+<body>
+
+<div id="root"> </div>
+	<script type="text/babel">
+
+	var List = React.createClass({
+
+		handleInput: function() {
+		  	var message = this.input.value;
```

Comment:
it seems to me that this is a good place to handle validation. Then you don't get into the creation mode at all, if the message is not valid